ruby/vampires.rb

```diff
@@ -0,0 +1,66 @@
+puts "How many employees will be processed?"
+employee = gets.to_i
+
+vampire = 0
+
+#Loop until employee processing is met.
+until vampire == employee 
+
+	puts "What is your name?"
+	name1 = gets.chomp
+    	if name1 == "Drake Cula"
+    		name2 = false
+    	elsif name1 == "Tu Fang"
+    		name2 = false
+    		else name2 = true
+    	end
+  
+    puts "How old are you? What year were you born?"
+	age = gets.to_i
+	birth_year = gets.to_i
+  		if birth_year + age == 2016
```

Comment:
Consider checking the birth year agains the Ruby time class `Time.now.year` this makes the code more dynamic and allows you the freedom of not having to rewrite the file with hardcoded values as years change.